src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/EnvelopeProcessorValidationTest.java

```diff
@@ -78,22 +85,75 @@ public void should_throw_exception_when_zip_file_has_mismatching_pdf() throws Ex
     }
 
     @Test
-    public void should_throw_exception_when_ocr_data_is_missing_for_document_type_sscs1() throws Exception {
-        ZipFileProcessingResult processingResult = processZip("zipcontents/missing_ocr_data");
-        InputEnvelope envelope = EnvelopeCreator.getEnvelopeFromMetafile(processingResult.getMetadata());
+    public void should_throw_exception_when_required_documents_are_missing() throws Exception {
+        InputEnvelope envelope = inputEnvelope(
+            "SSCS",
+            Classification.NEW_APPLICATION,
+            asList(
+                scannableItem(InputDocumentType.OTHER, emptyMap()), // no 'SSCS1' documents
+                scannableItem(InputDocumentType.CHERISHED, emptyMap())
+            )
+        );
 
         Throwable throwable = catchThrowable(() ->
             EnvelopeValidator.assertEnvelopeContainsOcrDataIfRequired(envelope)
         );
 
         assertThat(throwable).isInstanceOf(OcrDataNotFoundException.class)
-            .hasMessageContaining("No scannable items found with ocr data and document type SSCS1");
+            .hasMessageContaining("No documents");
     }
 
     @Test
-    public void should_not_throw_exception_when_ocr_exists_for_document_type_other() throws Exception {
-        ZipFileProcessingResult processingResult = processZip("zipcontents/ocr_data_not_required");
-        InputEnvelope envelope = EnvelopeCreator.getEnvelopeFromMetafile(processingResult.getMetadata());
+    public void should_throw_exception_when_document_type_for_new_applications_is_not_configured_for_given_jurisdiction() throws Exception {
```

Comment:
At some point we should split those tests by methods that are tested

Reply:
Or even have a separate class for every validation (for easier navigation)

Reply:
Yes, that's exactly the kind of split I had in mind